app/views/books/index.html.erb

```diff
@@ -1,24 +1,31 @@
-<h1>Bookstore</h1>
 <p>
-  <%= form_tag(root_path, method: :get) do %>
-    <%= text_field_tag :search %>
-    <%= submit_tag "Search" %>
+  <%= form_tag books_path, method: :get, class: 'form-inline' do %>
+    <div class='form-group'>
+      <%= text_field_tag :search, params[:search], class: 'form-control' %>
+    </div>
+    <%= submit_tag "Search", class: 'btn btn-default' %>
   <% end %>
-<table>
+</p>
+<%= link_to "Sort by Most Popular", root_path(sort: "times_purchased") %></p>
+<table class='table'>
   <thead>
     <tr>
       <th>Title</th>
       <th>Author</th>
       <th>Published Date</th>
       <th>Price</th>
+      <th>Times Purchased</th>
     </tr>
   <% @books.each do |book| %>
     <tr>
       <td><%= book.title %></td>
       <td><%= book.author %></td>
       <td><%= book.published_date %></td>
-      <td><%= book.price_cents %></td>
+      <td><%= book.price.format %></td>
+      <td><%= book.times_purchased %></td>
```

Comment:
Nitpicky thing here but `<tr>` and `<td>` tags are much harder to style than other tags. For instance, you'd have a lot more flexibility on the front end if you did
```<% @books.each do |book| %>
     <ul>
       <li><%= book.title %></tli>
       <li><%= book.author %></li>
       <li><%= book.published_date %></li> ...```

And then once you had your table using `<li>`s instead, you could use (flex)[https://css-tricks.com/snippets/css/a-guide-to-flexbox/] to have the "table" look or whatever look you wanted.